src/AccessibilityInsights.SharedUx/Utilities/ExtensionMethods.cs

```diff
@@ -181,9 +185,9 @@ public static List<RuleResultViewModel> GetRuleResultsViewModelList(this Element
         /// </summary>
         /// <param name="window"></param>
         /// <returns></returns>
-        public static Point GetTopLeftPoint(this Window window)
+        public static System.Windows.Point GetTopLeftPoint(this Window window)
         {
-            Point topLeft = new Point();
+            System.Windows.Point topLeft = new System.Windows.Point();
```

Comment:
Could we add some using or alias statements so that this code doesn't need to include the namespaces? I find the namespaces pretty distracting.

Reply:
The problem is that we now have Point in 2 namespaces, so we need to be explicit about which one we're referencing. We could use an alias, but that might be even more distracting